### PR TITLE
RBAC Architecture Documentation

### PR DESCRIPTION
**Pull request by lpichler**
original on https://github.com/ManageIQ/guides/pull/312

Rendered:
https://github.com/lpichler/guides/blob/rbac_1/architecture/rbac.md

cc @gtanzillo @Fryguy @jrafanie

### CODE REVIEW

File: architecture/rbac.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+
+- **Tenancy**
+
+Tenancy ownership in ManageIQs is determined by first a direct ownership to a resource, and then applying certain "ancestor" or "descendant" rules depending on the type of resource being viewed.
+
+
+![Tenancy](images/tenancy.svg)
+
+For example, a member of a tenant can only see virtual templates owned by their tenant or owned by any parent tenants. This "ancestor" view is used to allow for sharing a virtual template across multiple child tenants. In contrast, a member of a tenant can only see virtual machine instances owned by their tenant or owned by any child tenants. This "descendant" view is used to allow the parent tenant to do accounting of all running virtual machines instances below them.
```
Comment (lpichler) [inline architecture/rbac.md]: For example, a member of a tenant can only see virtual templates owned by their tenant or owned by any parent tenants. This "ancestor" view is used to allow for sharing a virtual template ** across multiple child tenants**....

Shouldn't be here **across multiple parent tenants **?

**Comment by Fryguy** [comment]
You can said my commits into yours to get rid of the PAUSED (i.e. WIP)

**Comment by Fryguy** [comment]
This looks good to me as a first pass. I would be good with merging this and then we can do cleanup PRs later.  @lpichler Can you swuash the commits to remove the "PAUSED" stuff and then I can merge?

**Comment by lpichler** [comment]
> This looks good to me as a first pass. I would be good with merging this and then we can do cleanup PRs later. @lpichler Can you swuash the commits to remove the "PAUSED" stuff and then I can merge?

@Fryguy Done, thanks 👍

**Pull request merged**